5-WebApp-AuthZ/5-2-Groups/README.md

```diff
@@ -203,13 +295,41 @@ The following files have the code that would be of interest to you..
 1. UserProfile\Index.cshtml
     1. Has some client code that prints the signed-in user's information obtained from the [/me](https://docs.microsoft.com/en-us/graph/api/user-get?view=graph-rest-1.0), [/me/photo](https://docs.microsoft.com/en-us/graph/api/profilephoto-get?view=graph-rest-1.0) and [/memberOf](https://docs.microsoft.com/en-us/graph/api/user-list-memberof?view=graph-rest-1.0) endpoints.
 1. Startup.cs
-    1.  in the `ConfigureServices` method, replace the two following lines:
 
      ```CSharp
+   - at the top of the file, add the following using directive:
+
+      using Microsoft.Identity.Web;
```

Comment:
Microsoft.Identity.Web does not seem to be used in the code below?
But we might want to write that the code below explains what happens, but in reality the sample uses Microsoft.Identity.Web which is reusable?

Reply:
Updated as it si being used